store/sqlstore/utils_test.go

```diff
@@ -76,3 +76,26 @@ func TestSanitizeSearchTerm(t *testing.T) {
 	result = sanitizeSearchTerm(term, "*")
 	require.Equal(t, result, expected)
 }
+
+func TestFilterNonAlphaNumericWords(t *testing.T) {
```

Comment:
Test with other languages or unicode characters

Reply:
Added tests in Japanese & Chinese in [this commit](https://github.com/amarjeetanandsingh/mattermost-server/commit/47556c0719ea24c604cf590ad538e677a1240175)